src/components/Dropdown/Dropdown.types.ts

```diff
@@ -24,12 +30,33 @@ export type DropdownDataGrouped = {
     options: DropdownDataOption[];
 }
 
+export type DropdownPill = DataLanguageType[]
```

Comment:
You don't need to create custom type for a specific story. All Dropdown data must use `DropdownData` (`DropdownDaraOption[]` or `DropdownDataGrouped[]`) or `DropdownDataTree`